ocaml/xapi/xapi_vif_helpers.ml

```diff
@@ -301,3 +301,39 @@ let copy ~__context ~vm ~preserve_mac_address vif =
         warn "Ignoring exception raised while creating PVS_proxy when copying a VIF: %s"
           (Printexc.to_string e)) proxies;
   result
+
+let backend_of_bridge_network net =
```

Comment:
These two functions actually fit better in xapi_xenops because they return xenops metadata. It seems to me that you've put them here, because you are calling `get_backend` from `xapi_network_sriov_helpers.ml` as well, but I'm not sure that it has to.

Reply:
Fixed.